silo-core/deploy/input/sonic/woS_wS.json

```
@@ -0,0 +1,29 @@
+{
+    "deployer": "0x7461d8c0fDF376c847b651D882DEa4C73fad2e4B",
+    "hookReceiver": "CLONE_IMPLEMENTATION",
+    "hookReceiverImplementation": "GaugeHookReceiver.sol",
+    "daoFee": 1500,
+    "deployerFee": 0,
+    "token0": "woS",
+    "solvencyOracle0": "ERC4626_woS",
+    "maxLtvOracle0": "NO_ORACLE",
+    "interestRateModel0": "InterestRateModelV2.sol",
+    "interestRateModelConfig0": "bridgeETHv15",
+    "maxLtv0": 9000,
+    "lt0": 9500,
+    "liquidationTargetLtv0": 9400,
+    "liquidationFee0": 350,
+    "flashloanFee0": 0,
+    "callBeforeQuote0": false,
+    "token1": "wS",
+    "solvencyOracle1": "NO_ORACLE",
+    "maxLtvOracle1": "NO_ORACLE",
+    "interestRateModel1": "InterestRateModelV2.sol",
+    "interestRateModelConfig1": "bridgeETHv15",
```

```suggestion
    "interestRateModelConfig1": "bridgeETHv11",
```
Model changed yesterday late PM.

---

Done